libraries/AP_Soaring/AP_Soaring.cpp

```diff
@@ -125,21 +125,36 @@ const AP_Param::GroupInfo SoaringController::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("ENABLE_CH", 15, SoaringController, soar_active_ch, 0),
 
+    // @Param: EXIT_MODE
```

Comment:
the param "TYPE" is probably better here. What do you think @WickedShell @tridge ?